feat: Use functional integration importants for JS SDKs (#9056)
getsentry/sentry-docs: platform-includes/performance/enable-automatic-instrumentation/javascript.vue.mdx

To enable tracing, include the `BrowserTracing` integration in your SDK configuration options.

> After configuration, you will see both `pageload` and `navigation` transactions in the Sentry UI.

Reviewer: ```suggestion
After configuration, you'll see both `pageload` and `navigation` transactions in the Sentry UI.
```
